src/db/group.rs

```diff
@@ -24,7 +24,7 @@ pub struct Group {
     pub icon_id: Option<usize>,
 
     /// UUID for a custom group icon
-    pub custom_icon_uuid: Option<String>,
```

Comment:
@sseemayer we could consider removing the `_uuid` suffix, now that the struct member is correctly typed.

Reply:
I've been trying to replicate the internal XML document's structure and naming as much as possible, but agree that there might be a chance to improve ergonomics for the users of the crate by doing some gentle renaming and potentially even restructuring.